quelpa.el

```diff
@@ -167,6 +168,10 @@ quelpa cache."
 (defvar quelpa-recipe '(quelpa :repo "quelpa/quelpa" :fetcher github)
   "The recipe for quelpa.")
 
+(defvar package-alist)
+(defvar package--initialized)
+(declare-function package-load-all-descriptors "ext:package")
+
```

Comment:
quelpa already has top-level `(require 'package)`. Is this necessary?

Reply:
It seems not, I will remove this